src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/CCJStaffNotificationServiceTest.java

```diff
@@ -69,14 +90,34 @@ public void shouldSendEmailToExpectedRecipient() {
         assertThat(senderArgument.getValue()).isEqualTo(emailProperties.getSender());
     }
 
+    @Test
+    public void shouldSendAdmissionEmailWithExpectedContent() {
+        service.notifyStaffCCJRequestSubmitted(claimWithAdmission);
+
+        verify(emailService).sendEmail(senderArgument.capture(), emailDataArgument.capture());
+
+        assertThat(emailDataArgument.getValue()
+            .getSubject())
+            .startsWith("Civil money claims:")
```

Comment:
we should need to validate the in between text thats between start and end.

Reply:
Rest is dynamic bit which we can't assert on subject line